### DISCUSSION TITLE
Hiding WinUI3 window from Taskbar

### DISCUSSION THREAD

**Discussion by Tide** [dotMorten/WinUIEx#33] [General]
I am wanting to have the same effect as ShowInTaskbar = false.  I have been looking at your library and some posts online and have come up with this so far.

I found this post [https://stackoverflow.com/a/9404749](https://stackoverflow.com/a/9404749) and it says that if I create a popup toolwindow then change the window back to a normal window it will have it not show in the Taskbar.

I have been trying to get it working but have some issues.  Is this something you would want to add to WinUIEx?  Do you think this would work for a WinUI3 window?  I know very little about Pinvoke so I am just not sure.

**Comment by dotMorten** [comment]
You should be able to use the `SetIsShownInSwitchers` extension method to do this.
https://dotmorten.github.io/WinUIEx/api/WinUIEx.WindowExtensions.SetIsShownInSwitchers.html#WinUIEx_WindowExtensions_SetIsShownInSwitchers_Microsoft_UI_Xaml_Window_System_Boolean_
